river/optim/schedulers.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, loss: optim.losses.Loss, alpha=1e-4):
         self.alpha = alpha
 
         typw = math.sqrt(1.0 / math.sqrt(self.alpha))
-        initial_eta0 = typw / max(1.0, self.loss.gradient(-typw, 1.0))
+        initial_eta0 = typw / max(1.0, self.loss.gradient(True, -typw))
```

Comment:
Can you leave a note in the PR description explaining this change? I'm not sure I follow it.